scripts/qa_crowbarsetup.sh

```diff
@@ -2595,10 +2595,15 @@ function hacloud_configure_cluster_defaults()
         proposal_set_value pacemaker "$clustername" \
             "['attributes']['pacemaker']['stonith']['sbd']['watchdog_module']" "'softdog'"
     else
-        proposal_set_value pacemaker "$clustername" \
-            "['attributes']['pacemaker']['stonith']['mode']" "'libvirt'"
-        proposal_set_value pacemaker "$clustername" \
-            "['attributes']['pacemaker']['stonith']['libvirt']['hypervisor_ip']" "'$admingw'"
+        if [[ $mkclouddriver = "libvirt" ]]; then
+            proposal_set_value pacemaker "$clustername" \
+                "['attributes']['pacemaker']['stonith']['mode']" "'libvirt'"
+            proposal_set_value pacemaker "$clustername" \
+                "['attributes']['pacemaker']['stonith']['libvirt']['hypervisor_ip']" "'$admingw'"
+        else
+            proposal_set_value pacemaker "$clustername" \
+                "['attributes']['pacemaker']['stonith']['mode']" "'manual'"
```

Comment:
Thats a good question. none of the upstream drivers for s390x are supported in the SLE HA product, the only option that remains is sbd, which is already covered in the case above under $want_sbd=1.